plugins/browser-plugin-youtube-tracking/src/index.ts

```diff
@@ -0,0 +1,231 @@
+/*
+ * Copyright (c) 2021 Snowplow Analytics Ltd
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice, this
+ *    list of conditions and the following disclaimer.
+ *
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ * 3. Neither the name of the copyright holder nor the names of its
+ *    contributors may be used to endorse or promote products derived from
+ *    this software without specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+ * FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+import { UrlParameters, trackingOptionsParser } from './helperFunctions';
+import { YTError, YTPlayerEvent, YTState, YTStateEvent } from './constants';
+import { EventData, MediaTrackingOptions, TrackedPlayer, TrackingOptions } from './types';
+import { BrowserPlugin, BrowserTracker, dispatchToTrackersInCollection } from '@snowplow/browser-tracker-core';
+import { buildSelfDescribingEvent, CommonEventProperties, Logger, SelfDescribingJson } from '@snowplow/tracker-core';
+import { SnowplowEvent } from './snowplowEvents';
+import { MediaPlayerEvent } from './contexts';
+import { buildYouTubeEvent } from './buildYouTubeEvent';
+import { YTEvent } from './youtubeEvents';
+
+declare global {
+  interface Window {
+    onYouTubeIframeAPIReady?: Function;
+  }
+}
+
+let LOG: Logger;
+export const _trackers: Record<string, BrowserTracker> = {};
```

Comment:
nit: export hidden amongst other varaibles (and it's const with all the lets). Consider bumping to top of list.

Reply:
Looking at this again, it shouldn't be exported at all.